src/store/modules/recipes/__tests__/actions.test.ts

```diff
@@ -0,0 +1,270 @@
+import { DataStatus } from "@/enums";
+import { actions } from "../actions";
+import { Actions, Mutations, type State } from "../types";
+import type { ActionContext, ActionHandler, Store } from "vuex";
+import type { RootState } from "@/store/types";
+import { recipes as recipesData } from "@/data/data.json";
+
+const { FETCH_ALL_RECIPES, ADD_RECIPE, UPDATE_RECIPE, DELETE_RECIPE_BY_ID } =
+  Actions;
+
+describe("actions", () => {
+  const storeContext = {} as Store<RootState>;
+
+  describe("FETCH_ALL_RECIPES", () => {
+    let commit: typeof jest.fn;
+
+    const fetchAllRecipes = actions[FETCH_ALL_RECIPES] as ActionHandler<
+      State,
+      RootState
+    >;
+
+    beforeEach(() => {
+      commit = jest.fn();
+    });
+
+    afterEach(() => {
+      jest.restoreAllMocks();
+    });
+
+    it("commits set recipes mutation if there was no error", async () => {
+      await fetchAllRecipes.call(storeContext, {
+        commit,
+      } as unknown as ActionContext<State, RootState>);
+
+      expect(commit).toHaveBeenCalledWith(
+        Mutations.SET_DATA_STATUS,
+        DataStatus.PENDING
+      );
+      expect(commit).toHaveBeenCalledWith(Mutations.SET_RECIPES, recipesData);
+      expect(commit).toHaveBeenCalledWith(
+        Mutations.SET_DATA_STATUS,
+        DataStatus.FULFILLED
+      );
+      expect(commit).toHaveBeenCalledTimes(3);
```

Comment:
better to check how many times called `commit` in the first place. Because if `commit` is called more/less you don't need to go further in the test